### DISCUSSION TITLE
Co-exist with existing web server?

### DISCUSSION THREAD

**Discussion by scarlion1** [bluesky-social/pds#22] [Q&A] [answered]
I have a server already running httpd on port 80 and 443, is there a way I can run a PDS also and have them co-exist?  Thanks

**Comment by T0biii** [comment]
maybe with a ReversProxy
i am using traefik with docker as ReversProxy and also PDS in docker and got with this more Webservices running
https://github.com/T0biii/pds/blob/44b5d22cde4702c8b31049b8a8cf9b618e7dc9d7/compose.yaml#L16-L20

**Comment by bnewbold** [comment, marked as answer]
Yup, it should be possible to run the PDS along side other web services, if the routes starting with the prefix `/xrpc/` are reverse-proxied to the PDS on a different port. Or, use a separate domain name and have that "virtual server" redirect.

Depending on setup, you may also need to route some `/.well-known/` endpoints to the PDS for identity resolution to work. This isn't needed if you only use DNS resolution of domain handles for hosted accounts.
